sdk/resourcemanager/autocent.js

```diff
@@ -144,12 +144,16 @@ function writeMarkdown() {
                     } else if (aVerNums[1] < bVerNums[1]) {
                         return 1;
                     } else {
-                        var aPatchNums = aVerNums[2].split("-beta.");
-                        var bPatchNums = bVerNums[2].split("-beta.");
+                        var aPatchNums = a.split("-beta.");
+                        var bPatchNums = b.split("-beta.");
+                        // sort GA version before beta version
                         if (aPatchNums.length < bPatchNums.length) {
                             return -1;
                         } else if (aPatchNums.length > bPatchNums.length) {
                             return 1;
+                        } else if (aPatchNums.length > 1) {
+                          // sort according to beta minor version
+                          return parseInt(bPatchNums[1]) - parseInt(aPatchNums[1]);
```

Comment:
One thing I forgot... which of below is bigger?

1.2.0-beta.1
1.0.0-beta.3

Reply:
Should be 1.2.0-beta.1 and appears earlier? 

It should already be dealt with from line 142 to 144, where it compares second minor version number. 

The `else` block only deal with the last minor version. 

Theoretically the following could happen, but for now we seem always to have the third version number 0:
1.0.2
1.0.1
1.0.2-beta.1

Reply:
ok, then they are good.